utils/configuration.py

```diff
@@ -17,6 +17,8 @@ class Configuration:
     # contain in order to be seen as meaningful.
     cluster_accuracy_threshold: int
 
+    max_depth: int
```

Comment:
Could you document this a bit? I guess it is related to the tree?

Reply:
I should remove the max_depth config